resources/user-guides/mine-to-pool.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: static_page
+title: How to mine on a pool
+---
+# Selecting a pool
+
+There are many pools to choose from, a list is available at
+[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
```

Comment:
>Do not recommend certain mining pools, miners, etc. Instead, use placeholders. Perhaps give an example with pool.getmonero.org as a fictitious mining pool.

I hope mentioning moneropools.com is okay. I decided that it would be better than maintaining a list of pools on this site, and it would prevent people going onto dodgy sites they find on Google.